server/server.go

```diff
@@ -8,29 +8,44 @@ import (
 
 	"github.com/NYTimes/gziphandler"
 	"github.com/gorilla/mux"
+	"go.opentelemetry.io/contrib/instrumentation/github.com/gorilla/mux/otelmux"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
 
 	"github.com/kiali/kiali/business"
 	"github.com/kiali/kiali/config"
 	"github.com/kiali/kiali/log"
+	"github.com/kiali/kiali/observability"
 	"github.com/kiali/kiali/routing"
 )
 
 type Server struct {
 	httpServer *http.Server
 	router     *mux.Router
+	tracer     *sdktrace.TracerProvider
 }
 
 // NewServer creates a new server configured with the given settings.
 // Start and Stop it with the corresponding functions.
 func NewServer() *Server {
 	conf := config.Get()
-	// create a router that will route all incoming API server requests to different handlers
 
+	// create a router that will route all incoming API server requests to different handlers
 	router := routing.NewRouter()
+	var tracingProvider *sdktrace.TracerProvider
+	if conf.Server.Observability.Tracing.Enabled {
+		log.Debugf("Tracing Enabled. Initializing tracer with collector url: %s", conf.Server.Observability.Tracing.CollectorURL)
```

Comment:
Minor, I'd like to see this log entry always, perhaps moving to "Info" can be a good idea.